DEVELOPERS

```diff
@@ -215,3 +216,28 @@ If in doubt, reach out on IRC or via a "question" Issue on GitHub.
   - no scraping HTML!
 - New code should be covered by unit-tests
 
+
+Creating a Release
+==================
+
+- Install the tools required:
+   - pip install --editable .[build]
```

Comment:
Consider invoking `pip` via `python -m pip` so that you're always using the `pip` from that python's environment (and not, e.g., the system one).